poppy/accel_math.py

```diff
@@ -221,7 +221,7 @@ def fft_2d(wavefront, forward=True, normalization=None, fftshift=True):
     _log.debug("using {2} FFT of {0} array, FFT_direction={1}".format(
         str(wavefront.shape), 'forward' if forward else 'backward', method))
 
-    if (forward) and fftshift: # shift before backwards propagations (using forward FFT)
+    if fftshift: # shift before backwards propagations (using forward FFT)
```

Comment:
Very very minor: It would be nice to also update the comment text here to "shift before all propagations" instead of "shift before backwards propagations"